docs/HACKING.md

```diff
@@ -1,158 +1,162 @@
-configured by the ConfigMap controller, skipping the deletion of all the nodes.
 ```shell script
-hack/run-ci-e2e-test.sh -s -m 2 -c 1
+make deploy IMG=$OPERATOR_IMAGE
 ```
-Please note that you do not need to run `hack/olm.sh run` before `hack/run-ci-e2e-test.sh`.
 
-#### Running e2e tests on platform-agnostic infrastructure
+The above command will directly deploy all the resources required for the operator to run, as well as the WMCO pod.
 
-To run the WMCO e2e tests on a bare metal or other platform-agnostic infrastructure (platform=none), where there
-is no cloud provider specification, the desired number of Windows instances must be provisioned beforehand, and the
-instance(s) information must be provided to the e2e test suite through the `WINDOWS_INSTANCES_DATA` environment
-variable.
+#### Cleaning up a manual deployment  
 
-To deploy machines using the [infrastructure providers supported by WMCO](wmco-prerequisites.md#supported-cloud-providers-based-on-okdocp-version-and-wmco-version),
-refer to the specific provider documentation. See Windows instance [pre-requisites](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/byoh-instance-pre-requisites.md).
+To remove the installed resources:
 
-The information you need to collect from each Windows instance is:
-- the internal IP address
-- the Windows administrator username
+```shell script
+make undeploy
+```
 
-Export `WINDOWS_INSTANCES_DATA` as an environment variable with the corresponding [windows-instances ConfigMap](https://github.com/openshift/windows-machine-config-operator#adding-instances)
-data section, and a new `windows-instances` ConfigMap will be created during the execution of the e2e test suite.
+### Running the OLM hack script
 
-For example:
-```shell
-export WINDOWS_INSTANCES_DATA='
-data:
-  10.1.42.1: |-
-    username=Administrator
-'
+Deploy the operator lifecycle manager onto your cluster with the hack script.
+
+This command builds the operator image, pushes it to remote repository and uses OLM to launch the operator.
+
+```shell script
+hack/olm.sh run -k KUBE_SSH_KEY_PATH
 ```
-where `10.1.42.1` is the IP address and `Administrator` is the Windows' administrator username of the Windows instance.
+#### Cleaning up an OLM deployment
 
-After the `WINDOWS_INSTANCES_DATA` environment variable is set and exported you can run:
-```shell
-hack/run-ci-e2e-test.sh
+```shell script
+hack/olm.sh cleanup
 ```
 
-## Bundling the Windows Machine Config Operator
-This directory contains resources related to installing the WMCO onto a cluster using OLM.
+### Deploying using the official deployment method
+
+If you want to deploy the WMCO using the official deployment method, first follow the instructions on how to
+[build and deploy the operator using the OLM](#running-the-olm-hack-script), or
+[build the operator manually](#build-an-operator-image) and [deploy](#deploying-the-operator-manually)
+
+### Generate new bundle manifests
```

Comment:
Why are this is a standalone header? Same question applies to:

- Create a bundle image
- Create a CatalogSource using the index
- Creating a new operator index

All of these are for creating your own custom CatalogSource for WMCO instead of using `redhat-operators` or `community-operators`.